### PR TITLE
Post-Brexit Secondary Component

### PR DESCRIPTION
**Pull request by sumeetadur**
- Replace poll tracker with article/currency chart (based on flag)
- Rename title to 'Brexit'
- Slightly adjusted design

screen shot 2016-07-08 at 11 52 14

### CODE REVIEW

File: server/routes/front-page.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ import * as components from '@financial-times/n-section';
 
 import { getData } from '../libs/graphql-poller';
 import { getBrexitLiveData } from '../libs/brexit-live-poller';
-import { getBrexitSecondaryResultsData } from '../libs/brexit-secondary-results';
+import { getBrexitSecondaryLiveData } from '../libs/brexit-secondary-live-poller';
 import BrexitLive from '../../shared/components/brexit-live/brexit-live';
 import BrexitLink from '../../shared/components/brexit-link/brexit-link';
-import BrexitSecondaryResults from '../../shared/components/brexit-secondary-results/brexit-secondary-results';
```
Comment (ironsidevsquincy) [inline server/routes/front-page.js]: Is this file still being used?

File: server/routes/front-page.js [hunk 2]
```diff
@@ -2,10 +2,10 @@ import * as components from '@financial-times/n-section';
 
 import { getData } from '../libs/graphql-poller';
 import { getBrexitLiveData } from '../libs/brexit-live-poller';
-import { getBrexitSecondaryResultsData } from '../libs/brexit-secondary-results';
```
Comment (ironsidevsquincy) [inline server/routes/front-page.js]: Is this file still being used?

Comment (sumeetadur) [inline server/routes/front-page.js]: No these aren't being used right now

**Comment by ironsidevsquincy** [comment]
Apart from removing unused files, 👍

**Pull request merged**